include/swift/AST/DiagnosticsSema.def

```diff
@@ -2662,6 +2662,8 @@ ERROR(inheritance_from_cf_class,none,
 ERROR(inheritance_from_objc_runtime_visible_class,none,
       "cannot inherit from class %0 because it is only visible via the "
       "Objective-C runtime", (Identifier))
+ERROR(composition_of_protocol_and_AnyObject,none,
```

Comment:
Also, normally diagnostic ids are all lower case so this would be `composition_of_protocol_and_anyobject` :)

Reply:
Noted!